app/scripts/cartodb/categories-service.js

```diff
@@ -0,0 +1,79 @@
+
+(function () {
+    'use strict';
+
+    /* ngInject */
+    function Categories ($log, $q, Config) {
+
+        var categories = null;
+        var categoriesQuery = [
+            'SELECT COUNT(*), SECTION, project_category, sub_category FROM master_datalist',
+            'GROUP BY SECTION, project_category, sub_category',
+            'ORDER BY SECTION, project_category, sub_category'
```

Comment:
Doesn't the default ordering match the group by columns? Being explicit is fine, though.